### ISSUE TITLE
sensitive .jsons

### ISSUE THREAD

**Issue by rikai** [gblosser42/TS-Dice#1]
Just thought i'd point out that you missed the auth.json here too

**Comment by gblosser42** [comment]
Yeah, that's an issue with the people who made the node module for discord. They have already corrected it on their end, removing the sensitive data and changing the credentials.
